app/database/models.py

```diff
@@ -262,6 +272,56 @@ class OAuthCredentials(Base):
     owner = relationship("User", back_populates=__tablename__, uselist=False)
 
 
+class Task(Base):
+    __tablename__ = "tasks"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String)
+    content = Column(String)
+    is_done = Column(Boolean, nullable=False)
+    is_important = Column(Boolean, nullable=False)
+    date_time = Column(DateTime, nullable=False)
```

Comment:
Try to give more specific variable names.
is it the creation date, or maybe the deadline, it is hard to tell.

Reply:
it's not too much my model to mess with